src/components/Card/Card.js

```diff
@@ -6,15 +6,17 @@ import StyledButton from '../../../styles/StyledButton';
 import StyledFlex from '../../../styles/StyledFlex';
 import StyledCard from './StyledCard';
 
-export default function Card() {
-	const myVehicles = useStore(state => state.myVehicles);
+export default function Card({ data }) {
 	const toggleDeleteState = useStore(state => state.toggleDeleteState);
 	const setID = useStore(state => state.setID);
 	const setVehicleName = useStore(state => state.setVehicleName);
+	const sortedVehicles = data.sort((a, b) => b.favorite - a.favorite);
```

Comment:
I like how you sort your vehicles. I would have probably used two filter-functions but your solution is shorter and quite smart ;)

Reply:
Thanks :) Took me a while to figure this out :D